explainer.md

```diff
@@ -18,96 +18,107 @@ permanent "block" policies, or
-   that was unrelated to the request. Given this uncertainty, user agents rely
-   on common placement of the permission prompt, usually in the top-left of the
-   page. Even in the best case, this has the unfortunate effect of shifting the
-   point to which users need to pay attention from the thing they clicked on to
-   some distant part of the user agent's UI (see example 2). User agents could
-   make better decisions and provide better prompts if they could make
-   well-founded assumptions about the nature of the user's interaction in the
-   content area, and the user's current area of focus.
-
-   ![](images/image2.png) \
-   _Example 2. An example where the permission prompt is far away from the user's
-   current area of focus. The permission prompt was triggered because the user has
-   just clicked on the crosshair icon in the bottom right, but the prompt is easy
-   to miss since it's on the opposite side of the page._
-
-1. **Regret**: When users decline to provide a capability to a given site, it
-   seems quite reasonable for user agents to suppress that site's future
-   requests for the same capability in order to avoid annoyance or abuse. That
-   said, users change their minds for good reasons, but sometimes struggle to
-   understand how to express that new decision to the user agent (see example
-   3). In these cases, the user agent's desire to protect the user backfires,
-   leaving the user confused about how to make the site they want to use work
-   the way they want it to.User agents could make better decisions and provide
-   better re-prompt UI if they could make well-founded assumptions about the
-   nature of the user's interaction in the content area, and the user's intent.
-
-   ![](images/image3.png) \
-   _Example 3. An example where the user previously blocked camera and microphone
-   access, but has now just expressed a strong intention to re-enable them by clicking
-   the unmute buttons. Because the user agent has no insight into this interaction
-   in the content area, it is compelled to respect the user's previous decision.
-   Especially in a stressful scenario such as an important presentation, users will
-   struggle to navigate the settings surfaces to change the permission decision._
-
-Overall, attempts to optimize the trade-off between usability and interruptions
-have hit practical limits because, fundamentally, user agents
+There are four main challenges with the status quo:
+
+1.  **Insufficiency of existing mitigations** The present day permissions spam
+    and abuse mitigation approach has an architectural upper bound on user
+    protection because the model relies on the website to choose when to trigger
+    the permission request prompt rather than capturing a reliable signal of
+    user intent. Requiring a user gesture for the Permission API (or similar)
+    does not solve this problem as there are many ways of tricking a user into
+    providing such a gesture.
```

Comment:
Note that this is not a function of the Permission API, but it is relate to the web's permission model (which is owned by the permission spec... it might be more accurate to say:  

```suggestion
    user intent. Requiring a user gesture to [request permission to use a powerful feature](https://www.w3.org/TR/permissions/#dfn-request-permission-to-use) (or similar)
    does not solve this problem as there are many ways of tricking a user into
    providing a so called "[activation triggering input event](https://html.spec.whatwg.org/#activation-triggering-input-event)" (i.e., a user gesture) .
```

Reply:
Accepted